<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Текстовый редактор</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="documentViewer">
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actCreate"/>
    <addaction name="actOpen"/>
    <addaction name="actSave"/>
    <addaction name="actSaveAs"/>
    <addaction name="actHelp"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Режим</string>
    </property>
    <addaction name="actReadOnly"/>
   </widget>
   <widget class="QMenu" name="menuAssignButton">
    <property name="title">
     <string>Назначить клавиши</string>
    </property>
    <addaction name="actCreateAssign"/>
    <addaction name="actOpenAssign"/>
    <addaction name="actSaveAssign"/>
    <addaction name="actSaveAsAssign"/>
    <addaction name="actExitAssign"/>
   </widget>
   <widget class="QMenu" name="menuDecoration">
    <property name="title">
     <string>Оформление</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Язык</string>
     </property>
     <addaction name="actRu"/>
     <addaction name="actEn"/>
    </widget>
    <widget class="QMenu" name="menuApplication">
     <property name="title">
      <string>Тема</string>
     </property>
     <addaction name="actWhite"/>
     <addaction name="actDark"/>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="menuApplication"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMode"/>
   <addaction name="menuAssignButton"/>
   <addaction name="menuDecoration"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actCreate">
   <property name="text">
    <string>Создать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="actHelp">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="actReadOnly">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Только для чтения</string>
   </property>
  </action>
  <action name="actCreateAssign">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="actOpenAssign">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actSaveAssign">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actSaveAsAssign">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="actExitAssign">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actRu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="actEn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Английский</string>
   </property>
  </action>
  <action name="actWhite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Светлая</string>
   </property>
  </action>
  <action name="actDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Темная</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
